lib/logmatcher.c

```diff
@@ -354,9 +354,9 @@ _jit_pcre2_regexp(LogMatcherPcreRe *self, const gchar *re, GError **error)
       PCRE2_UCHAR error_message[128];
 
       pcre2_get_error_message(rc, error_message, sizeof(error_message));
-      msg_warning("Failed to JIT compile regular expression, you might want to use flags(disable-jit)",
-                  evt_tag_str("regexp", re),
-                  evt_tag_str("error", (gchar *) error_message));
+      msg_debug("Failed to JIT compile regular expression, you might want to use flags(disable-jit) to stop trying",
```

Comment:
I don't think this warning adds too much value, especially if you have an old config file you want to use with a new syslog-ng that does JIT but it's not available due to a SELinux restriction.

Adding the disable-jit for everything in this case is just extra work that does not give benefits.

Back when I implemented JIT support, it was still experimental, so adding the flag allowed the user to disable it in case of errors in the JIT compiler. But now it's a lot more reliable and having to add all those flags in the config is just not worth it.

Reply:
@MrAnno for some reason this response of mine doesn't make it into the PR front page.

Can you agree to this, can I resolve the comment?